src/math/bspline.hpp

```diff
@@ -11,6 +11,16 @@ namespace precice::math::bspline {
  * @pre splineDegree must be larger than 0.
  * @pre t must be within [ts.min; ts.max].
  */
-Eigen::VectorXd interpolateAt(Eigen::VectorXd ts, const Eigen::MatrixXd &xs, int splineDegree, double t);
+class bspline {
 
+public:
+  Eigen::VectorXd interpolateAt(Eigen::VectorXd ts, const Eigen::MatrixXd &xs, int splineDegree, double t, bool recomputeBuffer);
```

Comment:
```suggestion
  Eigen::VectorXd interpolateAt(Eigen::VectorXd ts, const Eigen::MatrixXd &xs, int splineDegree, double t);
```
Not needed, if we make `_recomputeBuffer` a member of `Bspline`.